docs/UserGuide.md

```diff
@@ -408,18 +411,21 @@ Format: `edit_archived STUDENT_ID [id/STUDENT_ID] [n/NAME] [e/EMAIL] [g/GRADE] [
 
 * Edits the details of an archived student with the specified `STUDENT_ID`.
 * At least one of the optional fields indicated within the square bracket must be provided.
-* The student with the specified `STUDENT_ID` must exists in the archived list.
-* `Student_ID` should **start with A follow by 7 digits and ends with a letter [A-Z]**.
-  * e.g. `A0123459X`
-* `NAME` is case-sensitive, eg. john doe is different from John Doe.
-* Email must have the correct domain (@u.nus.edu) and string length of 8 for the email username eg. e0000000@u.nus.edu
+* The `...` after the `GROUP_NAME` indicates that multiple `GROUP_NAME`s can be provided for editing.
+* The student with the specified `STUDENT_ID` must exist in the archived list.
+* `STUDENT_ID` should start with 'A', followed by 7 digits, and end with any capital letter. 
+    * e.g. `A0123459X`
+* `EMAIL` should be of the format `username@domain`. The `username` should start with **e**, followed by 7 digits. The `domain` should be **u.nus.edu**.
+    * e.g. `e0123456@u.nus.edu`
+* `NAME` is case-sensitive.
+    * e.g. `John Doe` is different from `john doe`
 * `GRADE` should be one of the valid grades: **[A+, A, A-, B+, B, B-, C+, C, D+, D, F]**.
 * Editing `GROUP` will overwrite existing `GROUP` entries.
-* Use of `GROUP` prefix `gp/` in an edit command without specifying `GROUP_NAME` will clear all existing `GROUP` of the student.
+* *Use of `GROUP` prefix `gp/` in an edit command without specifying `GROUP_NAME` will clear all existing `GROUP`s of the student!*
```

Comment:
there's two bullet points and ! here.

Reply:
![image](https://github.com/AY2324S2-CS2103T-T09-1/tp/assets/122243884/3df76366-f35f-4d4f-8ba4-42400f3414ed)
should be smtg like this.

Reply:
I wrote that myself to emphasize that you should be careful of not specifying any group. Also, the * is part of another * at the end, and forms a underline.